__tests__/infrastructure/TransactionService.spec.js

```diff
@@ -70,53 +64,174 @@ describe('Transaction Service', () => {
 			} = await TransactionService.getTransactionInfo('541B506C91003C248CF09C6FB3BC94D0A26B8197B76B2827DDE3AF3A0ED3E350');
 
 			// Assert:
-			expect(blockHeight).toEqual(mockTransferTransaction.transactionInfo.height);
-			expect(transactionHash).toEqual(mockTransferTransaction.transactionInfo.hash);
-			expect(effectiveFee).toEqual(mockTransactionEffectiveFee);
-			expect(timestamp).toEqual(mockBlockInfo.timestamp);
-			expect(status).toEqual(mockTransactionStatus.detail.code);
-			expect(confirm).toEqual(mockTransactionStatus.message);
-			expect(transactionBody).toEqual(mockCreateTransactionFromSDK.transactionBody);
+			expect(blockHeight).toEqual(transferTransaction.transactionInfo.height);
+			expect(transactionHash).toEqual(transferTransaction.transactionInfo.hash);
+			expect(effectiveFee).toEqual('0.001760');
+			expect(timestamp).toEqual(1646063763);
+			expect(status).toEqual(transactionStatus.detail.code);
+			expect(confirm).toEqual(transactionStatus.message);
+			expect(transactionBody).toEqual(transactionFromSDK.transactionBody);
 		});
-	});
 
-	describe('getTransactionList', () => {
-		it('return transactions', async () => {
+		it('returns partial transaction with max fee', async () => {
 			// Arrange:
-			const pageInfo = {
-				pageNumber: 1,
-				pageSize: 10
+			const transactionStatus = {
+				message: 'partial',
+				detail: {
+					code: 'Success'
+				}
 			};
 
-			const mockBlockInfo = {
-				height: 198327,
-				timestamp: 1646063763
-			};
+			const innerTransaction = TestHelper.mockTransaction({
+				height: 0,
+				timestamp: 0
+			});
+
+			const partialTransaction = TestHelper.createPartialAggregateTransaction([], [innerTransaction]);
 
-			const mockSearchTransactions = {
-				...pageInfo,
-				data: [
-					TestHelper.mockTransaction(mockBlockInfo)
-				]
+			const partialTransactionFromSDK = {
+				...partialTransaction,
+				deadline: 1665914320,
+				maxFee: '0.050000',
+				type: 16961,
+				innerTransactions: [
+					{
+						...innerTransaction,
+						deadline: '2022-02-28 17:55:31',
+						maxFee: 0,
```

Comment:
inner transactions should not have these fields ?

Reply:
double-check on Rest, you are right 👍🏼